addons/web/static/src/scss/layout_boxed.scss

```diff
@@ -59,13 +48,8 @@
                     border-bottom: 1px solid gray('700');
                 }
                 &.o_line_section td {
-                    background-color: rgba($o-company-primary-color, 0.7);
```

Comment:
We need to keep defaults for all background-color properties, just like we needed to keep the default font, otherwise some layouts degrade badly, e.g. the total price in the boxed layout is white on white.

Moreover, seeing that the color detection is automatic when you set a logo in the layout wizard, it's likely that many users will have non-default colors, and have several months of prod use based on that. For this reason I think we must achieve 0 degradation in the mono-company case. Otherwise, how are users going to guess that they need to force-update the web module?

I'm going to test keeping the original scss files intact, and only relying on higher priority CSS overrides. We can always cleanup that cruft in 13.1.